src/spdx_tools/spdx3/bump_from_spdx2/creation_information.py

```diff
@@ -23,10 +23,14 @@ def bump_creation_information(spdx2_creation_info: Spdx2_CreationInfo, payload:
     print_missing_conversion("creation_info.document_namespace", 0, "https://github.com/spdx/spdx-3-model/issues/87")
 
     # creation_info.external_document_refs -> spdx_document.imports
```

Comment:
Probably either delete this comment or update to "-> spdx_document.namespaces, spdx_document.imports".

In general I think we can delete these comments. In the beginning I added them to get a better overview on the migration but with the migration guide I don't think that we need them any longer.